yb-voyager/cmd/exportSchemaOptimizationReport.go

```diff
@@ -123,117 +143,131 @@ func NewAppliedShardingRecommendationChange(objectType string) *ShardingRecommen
 		description = "Sharding recommendations from the assessment have been applied to optimize data distribution and performance."
 	}
 
+	if !applied {
+		title = "Sharding Recommendations - Not Applied"
+		description = APPLIED_RECOMMENDATIONS_NOT_APPLIED_DESCRIPTION
+	}
 	return &ShardingRecommendationChange{
 		Title:             title,
 		Description:       description,
 		ShardedObjects:    make([]string, 0),
 		CollocatedObjects: make([]string, 0),
-		IsApplied:         true,
+		IsApplied:         applied,
 	}
 }
 
-// NewSchemaOptimizationReport creates a new SchemaOptimizationReport with the given metadata
-func NewSchemaOptimizationReport(voyagerVersion, dbName, dbSchema, dbVersion string) *SchemaOptimizationReport {
-	return &SchemaOptimizationReport{
-		VoyagerVersion:        voyagerVersion,
-		SourceDatabaseName:    dbName,
-		SourceDatabaseSchema:  dbSchema,
-		SourceDatabaseVersion: dbVersion,
+type SecondaryIndexToRangeChange struct {
+	Title                    string              `json:"title"`
+	Description              string              `json:"description"`
+	ReferenceFile            string              `json:"reference_file"`
+	ReferenceFileDisplayName string              `json:"reference_file_display_name"`
+	ModifiedIndexes          map[string][]string `json:"modified_indexes"`
+	IsApplied                bool                `json:"is_applied"`
+}
+
+func NewSecondaryIndexToRangeChange(applied bool, referenceFile string, displayFileName string, modifiedIndexes map[string][]string) *SecondaryIndexToRangeChange {
+	title := "Secondary Indexes to be range-sharded - Applied"
+	description := "The following secondary indexes were converted to range-sharded indexes. This helps in distributing the data evenly across the nodes and improves the performance of these indexes."
```

Comment:
I think it would be good to give more information here as to why we're defaulting to range. 

How about: 

"The following secondary indexes were converted to range-sharded indexes in YugabyteDB. This helps in distributing the data evenly across the nodes, and avoids potential hotspots that come with HASH-sharded indexes. (high % of NULLs, high% of particular value, low cardinalty)" 

we can link to those issues in voyager docs as well. 

Same for the notApplied case

Reply:
made the change, please take a look at the screen shot of report in description